public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -294,11 +295,31 @@ export default class CloudMonitoringDatasource extends DataSourceWithBackend<
     return query;
   }
 
-  interpolateProps<T extends Record<string, any>>(object: T, scopedVars: ScopedVars = {}): T {
+  interpolatePromQLQuery(value: string | string[], _variable: QueryVariableModel) {
+    if (isArray(value)) {
+      return value.join('|');
+    }
+    return value;
+  }
+
+  interpolateProps<T extends Record<string, any>>(
+    object: T,
+    scopedVars: ScopedVars = {},
+    formattingFunctions?: { [key: string]: Function | undefined }
+  ): T {
     return Object.entries(object).reduce((acc, [key, value]) => {
+      let interpolatedValue = value;
+      if (interpolatedValue && isString(value)) {
```

Comment:
```suggestion
      if (value && isString(value)) {
```
We should use the same var for clarity. Either `value` or `interpolatedValue`

Reply:
Nice catch 😊